source/_integrations/bmw_connected_drive.markdown

```diff
@@ -129,9 +129,18 @@ The `button.<your_vehicle>_find_vehicle` button requests the vehicle to update t
 
 </div>
 
-### Update the state / refresh from API
+## Update the state / refresh from API
```

Comment:
This should not be documented on a integration basis. Please revert this change.

Reply:
Argh, sorry I did not realise that this was an issue.  

Would a better alternative be a more general comment to say that homeassistant.update_entity can be used to force a refresh if needed, without going into detail, and maybe linking to https://www.home-assistant.io/docs/scripts/service-calls/#homeassistant-services ?

Reply:
@davet2001 Yeah I think we should maybe introduce a page separately on the matter even?